scripts/scil_visualize_seeds.py

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Visualize seeds where streamlines originate from
+in a tractogram
+"""
+
+import argparse
+
+from fury import window, actor
+from nibabel.streamlines import detect_format, TrkFile
```

Comment:
You should be able to use whatever streamline format you want. Not TRK specific

Reply:
We can't save seeds in other formats than `trk` I think ?

Reply:
I just checked and `tck` does not support the `data_per_streamline` attribute

Reply:
Completely right